app/models/hackbot/helpers.rb

```diff
@@ -0,0 +1,30 @@
+module Hackbot
+  module Helpers
+    extend ActiveSupport::Concern
+
+    def access_token
+      team.bot_access_token
+    end
+
+    def msg
+      return nil unless event[:type] == 'message'
```

Comment:
Why are you doing this? If `event[:type] != 'message'` event [:text] will be `nil` anyway?